Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
```

Comment:
* :no_entry_sign: You must use "/**" style comments for a file comment
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 058d2234937bad7136b9979d0a743f66</sub>